src/index.js

```diff
@@ -47,6 +48,8 @@ function toFragment({node, trees, lang, title, inline = false}) {
     .flatMap((c) => c);
 }
 
+const themeCache = new Map();
```

Comment:
Shouldn't this be called `optionsCache`?

Reply:
Yeah, this is an outdated name when I was hashing only the `theme` object, not entire options.l